test/services/SicCodeService.test.ts

```diff
@@ -0,0 +1,50 @@
+import axios from "axios";
+import { SicCodeService } from "../../src/services/SicCodeService";
+import { generateTestData } from "../TestData";
+import { StatusCodes } from "http-status-codes";
+
+jest.mock("axios");
+const mockedAxios = axios as jest.Mocked<typeof axios>;
+mockedAxios.create = jest.fn(() => mockedAxios);
+
+describe("SicCodeService", () => {
+  const sicCodeService = new SicCodeService();
```

Comment:
Sorry - I don't think this is _quite_ what I was expecting. I was thinking we could use the `beforeEach()` function (like what's used in the `config.test.ts` file) to create a new instance of the service before each unit test. This would provide a `clean slate` for each test so it couldn't have any knock on effect to other tests. Something similar to:
```
  let sicCodeService: SicCodeService;
  beforeEach(() => {
      sicCodeService = new SicCodeService();
  });
```